infra/README.md

```diff
@@ -66,8 +67,10 @@ Instructions:
 
    gcp_zone = "<gcp-zone>"
 
+   gcp_location = "<gcp-region>"
```

Comment:
@daniel-goldstein, do you recall why this was added? It seems to me that there is now to distinct variables that refer to the same thing: the region in which to deploy. Is that true? Any reason for me to not change all occurrences of gcp_location to gcp_region?

Reply:
Hm I don't recall, I believe you should be ok making that change, but I would do a grep through the code first.